packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -137,7 +266,7 @@ describe("Garbage Collection Tests", () => {
 
         it("Session expires for an existing container", async () => {
             const metadata: IContainerRuntimeMetadata =
-                { summaryFormatVersion: 1, message: undefined, sessionExpiryTimeoutMs: 10 };
+                { summaryFormatVersion: 1, message: undefined, sweepEnabled: true, sessionExpiryTimeoutMs: 10 };
```

Comment:
does it matter that sweepEnabled is true? I thought sessionExpiry was going to be on regardless so that we could turn on sweep.

Reply:
You're right, not necessary.  Vestige from partway through previous PR